runtime/doc/version9.txt

```diff
@@ -41642,9 +41642,6 @@ Improved visual highlighting.
 
 Python3 support in OpenVMS.
 
-Support for |fuzzy-matching| during |ins-completion| with the "fuzzy" and
-"fuzzycollect" values of the 'completeopt' setting
-
```

Comment:
You should just remove "fuzzycollect" from the sentence instead of deleting the whole sentence.

Reply:
oops sry.